slsa/github.go

```diff
@@ -63,12 +66,25 @@ type WorkflowParameters struct {
 // NewWorkflowRun returns a generic WorkflowRun based on the
 // github context without special knowledge of the build.
 func NewWorkflowRun(s []intoto.Subject, c github.WorkflowContext) WorkflowRun {
+	// Create the entrypoint from the repository URI @ workflow path in the
+	// event. `workflow` is not used from the github context because it includes
+	// the workflow name rather than the path.
+	// NOTE: ServerURL does not have a trailing slash and Repository does not
+	// have a leading slash.
+	entryPoint := fmt.Sprintf("%s/%s",
+		c.ServerURL,
+		path.Join(
+			c.Repository,
+			fmt.Sprintf("%s", c.Event["workflow"]),
```

Comment:
@MarkLodato @joshuagl Now that I look at this again, the entrypoint could maybe just be `c.Event["workflow"]` which is the the path to the user workflow within the repository pointed to by URI (line 88). We can maybe leave off the ServerURL and Repository part rather than a full URI. Any thoughts?

Reply:
Agreed. I like `entryPoint` being the path within the repo at `uri`

Reply:
Yes, being relative to the repo was the intention.

Reply:
Done.